### PR TITLE
Error puppy global error page

### PR DESCRIPTION
**Pull request by jfversluis**
If something happens, at least the user sees a cute puppy!

screen shot 2017-01-28 at 14 56 35

### CODE REVIEW

**Comment by jfversluis** [comment]
Good point!

Got it from another project and tried to find the origin but can't tell for sure. Maybe we should substitute it with a cute looking Tamarin to be on the safe side?

**Comment by Cheesebaron** [comment]
Yeah, some stock photo with a clear license so we don't step over someones toes.

**Comment by jamesmontemagno** [comment]
Could use any of my photos: https://twitter.com/motzmonkeys

**Comment by jfversluis** [comment]
You're right @jamesmontemagno and totally forgot about your account!
Went for the Death Star waffle!

File: src/Firehose.Web/Views/Home/Error.cshtml [hunk 1]
```diff
@@ -25,10 +25,10 @@
                 </div>
             }
 
-            <p>To make it a little bit better, here is a picture of a cute puppy!</p>
+            <p>To brighten your life, here is a picture of a Xamarin monkey eating a Deatch Star waffle!</p>
```
Comment (Cheesebaron) [inline src/Firehose.Web/Views/Home/Error.cshtml]: Death Star

File: src/Firehose.Web/Views/Home/Error.cshtml [hunk 2]
```diff
@@ -25,10 +25,10 @@
...
 
             <div class="text-center">
-                <img src="~/Content/img/puppy.png" title="Awww look at it, it is so cute" alt="Cute puppy to make it all better" />
+                <img src="~/Content/img/deathstar_monkey.jpg" class="img-rounded" title="Awww look at it, it is so cute" alt="Xamarin monkey eating a Deatch Star waffle to make it all better" />
```
Comment (Cheesebaron) [inline src/Firehose.Web/Views/Home/Error.cshtml]: Death Start

**Pull request merged**